demo-app/src/components/aruba/ArubaMap.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Box } from 'grommet';
+import { Grommet } from 'grommet-icons';
+import {
+  Cluster,
+  Controls,
+  Map,
+  Marker,
+  MarkerCluster,
+  Pin,
+} from 'grommet-leaflet-core';
+
+import { userLocation, formatLocationsToLatLng } from '../../utils/locations';
+import { generateArubaLocations } from './ArubaData';
+import { ClusterPopup } from '../../ClusterPopup';
+import { ArubaPin } from './ArubaPin';
```

Comment:
Can we add an `index.js` file in this repo so we can import for `'.'`?

Also, to keep things more generic, can we remove all the "Aruba" references and rename to "network"? (Including the pages/aruba.jsx, I forgot to make that one generic)

Reply:
Ended up changing file structure (see Matt's comments)